gcp_variant_transforms/beam_io/vcf_estimate_io_test.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2019 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for vcfio_header_io module."""
+
+import os
+import unittest
+
+import apache_beam as beam
+from apache_beam.testing.util import assert_that
+from apache_beam.testing.util import equal_to
+import apache_beam.io.source_test_utils as source_test_utils
+from apache_beam.testing.test_pipeline import TestPipeline
+from gcp_variant_transforms.beam_io.vcf_estimate_io import VcfEstimate
+from gcp_variant_transforms.beam_io.vcf_estimate_io import VcfEstimateSource
+from gcp_variant_transforms.beam_io.vcf_estimate_io import GetAllEstimates
+from gcp_variant_transforms.beam_io.vcf_estimate_io import GetEstimates
+from gcp_variant_transforms.testing import asserts
+from gcp_variant_transforms.testing import temp_dir
+from gcp_variant_transforms.testing import testdata_util
+
+
+def _construct_estimate(headers, records, file_name=None):
+  single_line_size = len(records[0].encode('utf8'))
+  total_record_size = sum([len(x.encode('utf8')) for x in records])
+  total_header_size = sum([len(x.encode('utf8')) for x in headers])
+  return VcfEstimate(
+      file_name=file_name,
+      estimated_line_count=float(total_record_size)/single_line_size,
+      samples=headers[-1].split()[9:],
+      size_in_bytes=total_record_size+total_header_size)
+
+def _get_estimate_from_lines(lines, file_name=None):
+  for i in range(len(lines)): # pylint: disable=consider-using-enumerate
+    if lines[i][0] != '#':
+      return _construct_estimate(lines[:i], lines[i:], file_name)
+
+class VcfEstimateSourceTest(unittest.TestCase):
+
+  # TODO(msaul): Replace get_full_dir() with function from utils.
```

Comment:
Why is this not fixed here?

Reply:
Hmm, I missed that when copying from header tests.

However, I'm not quite certain what's being asked for here - get_full_dir() is already part of utils... I've found the PR which introduces it ( #7 ), but it doesn't have any more detail. It's not immediately clear to me which part of the current behavior is problematic either. Any idea? If not, I can just remove the comment (but I'd rather understand what was the concern two years ago...)

Reply:
Sorry means nothing to me.  Maybe Allie knows.